en/focus_privacy_notice.md

```diff
@@ -1,24 +1,29 @@
-# Firefox Focus Privacy Notice
+# Firefox Focus and Klar Privacy Notice
 
-June 20, 2017
-{: datetime="2017-06-20" }
+October 6, 2021
+{: datetime="2021-10-06" }
 
-At Mozilla, we design products with your privacy in mind. This privacy notice explains what data we collect in Focus, why, and your choices.  
+At Mozilla, we design products with your privacy in mind. This privacy notice explains what data we collect in Focus and Klar, why, and your choices. Our statements about Firefox Focus apply to both Focus and Klar unless we say otherwise. We also adhere to the practices outlined in the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/) for how we receive, handle and share information we collect from Firefox Focus. 
  
-To learn more about how we receive and handle information, read the [Mozilla Privacy Policy](https://www.mozilla.org/privacy/).
-
 ## Things you should know
 
-Focus, by default, sends data to Mozilla to help us improve our product.
+Focus sends data to Mozilla to: 
```

Comment:
Since this doc now covers both Focus and Klar, should we make it clear that the default for Klar is to not send any data? That the collection is opt-in?